README.md

```diff
@@ -11,4 +11,54 @@ Tool to sync files to a remote machine, similar to the standard Linux
 uses a higher performing remote diffing approach based on CDC.
```

Comment:
Can you change the leading words of this paragraph (line 9) to form a complete sentence?

Reply:
Done.